### ISSUE TITLE
Improvements on the code base

### ISSUE THREAD

**Issue by TomasLife** [xtr4nge/FruityWifi#236]
[ @xtr4nge ]
In my point of view this is a great project for the Raspberry Pi, however I think that the code base needs to be re-done, in order to have a cleaner code and making the project "more modular".

Also, the interface is kind of buggy a the moment and some modules don't work well.

Perhaps we could use some micro PHP framework such as Slim and/or Lumen.

Finally, if any help is needed I think there are plenty of people available to make this project better, including myself.

To sum up, I think that this kind of improvement would allow the project to improve the  "on the go" user experience.

Thank you for your attention.
